mir-layout/src/main/resources/META-INF/resources/mir-layout/scss/common/forms.scss

```diff
@@ -139,7 +139,7 @@
 }
 
 .dateExtended_box .mir-fieldset-legend, .personExtended_box .mir-fieldset-legend {
-    font-size: 14px;
+    font-size: 1.1rem;
```

Comment:
we should never set font-size directly in such cases, use variables instead
perhaps $font-size-base is the right one here?

Reply:
I will give it a try maybe $font-size-base will be to small.

Reply:
In this case we should look which variable fits or set this value relative to $font-size-base